src/azure-cli/azure/cli/command_modules/deploymentmanager/custom.py

```diff
@@ -271,39 +275,68 @@ def cli_service_unit_update(
 def cli_step_create(
         cmd,
         resource_group_name,
-        step_name,
-        duration,
+        step_name=None,
+        step=None,
+        duration=None,
         location=None,
         tags=None):
 
-    waitStepProperties = WaitStepProperties(
-        attributes=WaitStepAttributes(duration=duration))
+    if (step is None and duration is None):
+        raise CLIError('usage error: specify either step or duration. If step is specified, it can either be a wait step or health check step.')  # pylint: disable=line-too-long
```

Comment:
Could wrap line like:
```
usage error: specify either step or duration.
If step is specified, it can either be a wait step or health check step.
```
And then remove **# pylint: disable=line-too-long**